sql-plugin/src/main/scala/org/apache/spark/sql/rapids/complexTypeCreator.scala

```diff
@@ -87,8 +87,15 @@ case class GpuCreateNamedStruct(children: Seq[Expression]) extends GpuExpression
     case Seq(name, value) => (name, value)
   }.toList.unzip
 
+  // Names will be serialized before Spark scheduling, and the returned type GpuScalar
+  // from GpuLiteral.columnarEval(null) can't be serializable, which causes
+  // `org.apache.spark.SparkException: Task not serializable` issue.
+  //
+  // And on the other hand, the calling for columnarEval(null) in the driver side is
+  // dangerous for GpuExpressions, we'll have to pull it apart manually.
   private lazy val names = nameExprs.map {
     case gl: GpuLiteral => gl.value
+    case GpuAlias(gl: GpuLiteral, _) => gl.value
```

Comment:
Technically Alias can nest arbitrarily, so I think we'd want to match on the result of calling something like what `GpuOverrides.extractLit` does but for `GpuLiteral` or what Spark's `AliasHelper.trimAliases` does, probably handling `GpuAlias` since it's already been translated at that point.

Reply:
Yeah, Good suggestion, from the safe side, I added the extractGpulit for this case.
Thx

Reply:
Done